### ISSUE TITLE
unexpected server response (200)

### ISSUE THREAD

**Issue by mbregg1** [yhsiang/irccloud-cli#17]
In the past day or so, I'm unable to connect, and am seeing the following:

```
mike@micro:~$ ic -P . -v
[SUCCESS] Successfully obtained authentication token
[SUCCESS] Successfully logged in as user@email.com

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: unexpected server response (200)
    at ClientRequest.<anonymous> (/usr/local/lib/node_modules/irccloud-cli/node_modules/ws/lib/WebSocket.js:585:19)
    at ClientRequest.g (events.js:180:16)
    at ClientRequest.emit (events.js:95:17)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (http.js:1688:21)
    at HTTPParser.parserOnHeadersComplete [as onHeadersComplete] (http.js:121:23)
    at CleartextStream.socketOnData [as ondata] (http.js:1583:20)
    at CleartextStream.read [as _read] (tls.js:511:12)
    at CleartextStream.Readable.read (_stream_readable.js:323:10)
    at EncryptedStream.write [as _write] (tls.js:366:25)
    at doWrite (_stream_writable.js:226:10)
```

It seems that irccloud may have changed something in the past few days, but I'm not sure what.

**Comment by yhsiang** [comment]
Fixed with 0.3.1.

Please use new version

**Comment by PR0r** [comment]
thanks ;)
